templates/ci/python-django.yml

```diff
@@ -57,33 +25,27 @@ phases:
   - script: python -m pip install --upgrade pip setuptools wheel
     displayName: 'Install dependencies'
 
-  - script: python -m pip install -r requirements.txt
+  - script: pip install -r requirements.txt
```

Comment:
This should be fixed on Linux agents now.  I left it in `python -m pip install --upgrade pip` above since Windows needs `python -m` so as not to lock pip.exe.

Reply:
:+1: